tests/unit/test_config.py

```diff
@@ -381,8 +381,10 @@ def test_can_load_python_version():
         expected_runtime = 'python3.10'
     elif minor <= 11:
         expected_runtime = 'python3.11'
-    else:
+    elif minor <= 12:
```

Comment:
nit; This test could probably be improved. We're no longer testing on 2.7 (or 3.7). I don't think there's any platform that still allows you to deploy those either. We could probably remove the major version 2 check and then simplify the test to just:
```python
    c = Config('dev')
    major, minor = sys.version_info[0], sys.version_info[1]
    assert c.lambda_python_version == f"python{major}.{minor}"
```